pkg/states/state_manager.go

```diff
@@ -0,0 +1,77 @@
+package states
+
+import "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+
+func Active(userSignup *v1alpha1.UserSignup) bool {
+	return Approved(userSignup) &&
+		!VerificationRequired(userSignup) &&
+		!Deactivated(userSignup)
+}
+
+func Approved(userSignup *v1alpha1.UserSignup) bool {
+	return contains(userSignup.Spec.States, v1alpha1.UserSignupStateApproved)
+}
+
+func SetApproved(userSignup *v1alpha1.UserSignup, val bool) {
+	setState(userSignup, v1alpha1.UserSignupStateApproved, val)
+
+	if val {
+		setState(userSignup, v1alpha1.UserSignupStateVerificationRequired, false)
+	}
+}
+
+func VerificationRequired(userSignup *v1alpha1.UserSignup) bool {
```

Comment:
the same here with the name of the function - it makes more sense to call `IsVerificationRequired` when the function returns a boolean

Reply:
See above comment about Golang naming recommendations.

Reply:
:+1: